internal/start/start.go

```diff
@@ -43,9 +44,39 @@ func suggestUpdateCmd(serviceImages map[string]string) string {
 	return cmd
 }
 
+func validateExcludedContainers(excludedContainers []string) {
+	// Validate excluded containers
+	validContainers := ExcludableContainers()
+	var invalidContainers []string
+
+	for _, e := range excludedContainers {
+		found := false
+		for _, v := range validContainers {
+			if strings.EqualFold(e, v) {
+				found = true
+				break
+			}
+		}
+		if !found {
+			invalidContainers = append(invalidContainers, e)
+		}
+	}
+
+	if len(invalidContainers) > 0 {
+		// Sort the names list so it's easier to visually spot the one you looking for
+		sort.Strings(validContainers)
+		warning := fmt.Sprintf("%s The following container names are not valid to exclude: %s\nValid containers to exclude are: %s\n",
+			utils.Yellow("Warning:"),
+			utils.Aqua(strings.Join(invalidContainers, ", ")),
+			utils.Aqua(strings.Join(validContainers, ", ")))
+		fmt.Fprint(os.Stderr, warning)
+	}
+}
```

Comment:
Let's move this check to `cmd/start.go` since it's closely related to validating command line flag.